code/modules/reagents/dispenser/dispenser2.dm

```diff
@@ -151,17 +151,7 @@
 		var/label = href_list["dispense"]
 		if(cartridges[label] && container && container.is_open_container())
 			var/obj/item/weapon/reagent_containers/chem_disp_cartridge/C = cartridges[label]
-			var/mult = 1 + (-0.5 + round(rand(), 0.1))*(user.skill_fail_chance(core_skill, 0.3, SKILL_ADEPT))
-			C.reagents.trans_to(container, amount*mult)
-			var/contaminants_left = rand(0, max(SKILL_ADEPT - user.get_skill_value(core_skill), 0)) * can_contaminate
```

Comment:
Fully against this.

People without the complex devices (it is a complex device) skill can now go mad zoomies on the machine and never fuck up.

A: Not realistic.
B: Defeats purpose of doctors.

Reply:
Although it isn't realistic, I don't think anyone actually likes this feature at all. It just isn't fun to click "All" 7 times when you want to make something simply because you forgot to set Complex Devices to 3.